app/provider/geo_provider.py

```diff
@@ -1,22 +1,21 @@
-import boto3
-from app.geolocation.factory import Factory as GeolocationFactory
-from app import conf
 import logging
+import os
```

Comment:
is this import used?

Reply:
good catch, it is not, dropping the import